ownCloudAppFramework/AppLock Settings/AppLockSettings.m

```diff
@@ -80,7 +82,14 @@ + (OCClassSettingsMetadataCollection)classSettingsMetadata
 			OCClassSettingsMetadataKeyStatus	: OCClassSettingsKeyStatusAdvanced,
 			OCClassSettingsMetadataKeyCategory	: @"Passcode"
 		},
-
+		
+		OCClassSettingsKeyRequiredPasscodeDigits : @{
```

Comment:
Likewise, this should probably be for `OCClassSettingsKeyPasscodeEnforcedByDevice` here, too.